vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -129,8 +139,22 @@ XamlView NativeUIManager::reactPeerOrContainerFrom(xaml::FrameworkElement fe) {
   return nullptr;
 }
 
+// Previous versions of react-native-windows did not configure Yoga to use "LegacyStretchBehaviour"
+// We now set it, but to allow apps a period to transition, we are adding a property that can be set to
+// maintain the previous behavior.
+winrt::Microsoft::ReactNative::IReactPropertyName LegacyStretchBehaviourProperty() noexcept {
+  static winrt::Microsoft::ReactNative::IReactPropertyName propName =
+      winrt::Microsoft::ReactNative::ReactPropertyBagHelper::GetName(
```

Comment:
>winrt [](start = 6, length = 5)

Consider using the ReactPropertyId instead.
We can return its Hanlder().

Reply:
refactored a bunch.